tests/unit/experimental/test_config.py

```diff
@@ -224,7 +225,7 @@ def test_config_class():
     assert isinstance(cfg['output_dir'], Path)
     assert isinstance(cfg['auxiliary_data_dir'], Path)
 
-    from esmvalcore._config import CFG as CFG_DEV
+    from esmvalcore._config._config import CFG as CFG_DEV
```

Comment:
Has the location of this import moved in this pull request?

Reply:
No, as you mentioned in the comment above, this code is unsafe. This test depended on the config developer to be set in `esmvalcore._config.CFG` in `test_data_finder.py`. The location of `_config` was changed in #939.

Reply:
But why are the tests passing with the old import, if this was changed 2 months ago?

Reply:
Because `test_data_finder` sets the variable:
`test_data_finder.py`: `esmvalcore._config.CFG = esmvalcore._config.read_config_developer_file()`
`test_config.py`: `from esmvalcore._config import CFG as CFG_DEV`